lib/ps_set.fz

```diff
@@ -36,7 +36,7 @@
 #
 ps_set
   (K type : has_total_order,
-   psm ref ps_map K unit,  # NYI: #167: 'ref' due to lack of 'like this'
+   psm ps_map K unit,
```

Comment:
My original idea was to use `psm ps_set.this.type`, not sure if this works, though.